web/components/contract/contract-details.tsx

```diff
@@ -120,74 +127,157 @@ export function ContractDetails(props: {
   const user = useUser()
   const isCreator = user?.id === creatorId
-  const [open, setOpen] = useState(false)
   const { width } = useWindowSize()
-  const isMobile = (width ?? 0) < 600
+  const isMobile = getIsMobile()
+
+  return (
+    <Col>
+      <Row>
+        <Avatar
+          username={creatorUsername}
+          avatarUrl={creatorAvatarUrl}
+          noLink={disabled}
+          size={9}
+          className="mr-1.5"
+        />
+        {!disabled && (
+          <div className="absolute mt-3 ml-[11px]">
+            <MiniUserFollowButton userId={creatorId} />
+          </div>
+        )}
+        <Col className="text-greyscale-6 ml-2 flex-1 flex-wrap text-sm">
+          <Row className="w-full justify-between ">
+            {disabled ? (
+              creatorName
+            ) : (
+              <UserLink
+                className="my-auto whitespace-nowrap"
+                name={creatorName}
+                username={creatorUsername}
+                short={isMobile}
+              />
+            )}
+          </Row>
+          <Row className="text-2xs text-greyscale-4 gap-2 sm:text-xs">
+            {(!!closeTime || !!resolvedDate) && (
+              <Row className="items-center gap-1">
+                {resolvedDate && resolutionTime ? (
+                  <>
+                    <DateTimeTooltip
+                      text="Market resolved:"
+                      time={resolutionTime}
+                    >
+                      <Row>
+                        <div>resolved&nbsp;</div>
+                        <b>{resolvedDate}</b>
+                      </Row>
+                    </DateTimeTooltip>
+                  </>
+                ) : null}
+
+                {!resolvedDate && closeTime && user && (
+                  <Row>
+                    <div>Closes&nbsp;</div>
+                    <EditableCloseDate
+                      closeTime={closeTime}
+                      contract={contract}
+                      isCreator={isCreator ?? false}
+                    />
+                  </Row>
+                )}
+              </Row>
+            )}
+            {!isMobile && (
+              <MarketGroups
+                contract={contract}
+                isMobile={isMobile}
+                disabled={disabled}
+              />
+            )}
+          </Row>
+        </Col>
+        <div className="mt-0">
+          <ExtraContractActionsRow contract={contract} />
+        </div>
+      </Row>
+      {/* GROUPS */}
+      {isMobile && (
+        <div className="mt-2">
+          <MarketGroups
+            contract={contract}
+            isMobile={isMobile}
+            disabled={disabled}
+          />
+        </div>
+      )}
+      {/* {user && (
+        <>
+          <Row className="hidden items-center gap-1 md:inline-flex">
+            <DatabaseIcon className="h-5 w-5" />
+            <div className="whitespace-nowrap">{volumeLabel}</div>
+          </Row>
+        </>
+      )} */}
```

Comment:
nit: it's almost always not a great idea to commit commented out code.
Better to just remove. Someone could always dig it up in the git history.

Reply:
Done